opentelemetry-sdk/CHANGELOG.md

```diff
@@ -215,6 +215,24 @@ metadata, a feature introduced in version 0.1.40. [#2418](https://github.com/ope
   "rust.opentelemetry.io/sdk/tracer"
   [#2486](https://github.com/open-telemetry/opentelemetry-rust/pull/2486)
 
+- **Breaking** Introduced `experimental_async_runtime` feature for runtime-specific traits.
+  - Runtime-specific features (`rt-tokio`, `rt-tokio-current-thread`, and `rt-async-std`) 
+  now depend on the `experimental_async_runtime` feature.
+  - Custom runtime implementations must enable the `experimental_async_runtime` feature.
+  - For users needing to enable the runtime related features
```

Comment:
For most users, no action is required. Enabling runtime features such as rt-tokio, rt-tokio-current-thread, or rt-async-std will automatically enable the experimental_async_runtime feature.

If you're implementing a custom runtime, you must explicitly enable the experimental_async_runtime feature in your Cargo.toml and implement the required Runtime trait.

^ alternate wording suggestion.

Reply:
update the changelog as above.